docs/src/IceNucleation.md

```diff
@@ -42,3 +42,103 @@ Both parameters are dependent on aerosol properties and temperature.
     For a ``f_i`` values above 0.08 or ``S_i`` between 1.35 and 1.5,
     freezing occurs in a different ice nucleation mode
     (either a second deposition or other condensation type mode).
+
+## ABIFM for Sulphuric Acid Containing Droplets
+Water Activity-Based Immersion Freezing Model (ABFIM) is a method of parameterizing immersion freezing inspired by the time-dependent classical nucleation theory. The nucleation rate coefficient, ``J``, can be determined by the water activity, ``a_w``. This parameterization follows [KnopfAlpert2013](@cite), [Koop2002](@cite), [MurphyKoop2005](@cite), and [Luo1995](@cite). In this model, aerosols are assumed to contain an insoluble and soluble material. When immersed in water, the soluble material diffuses into the liquid water to create a sulphuric acid solution.
+
+Using empirical coefficients, ``m`` and ``c``, from [KnopfAlpert2013](@cite), the heterogeneous nucleation rate coefficient in units of ``cm^{-2}s^{-1}`` can be determined by the linear equation
+```math
+\begin{equation}
+  log_{10}J_{het} = m \Delta a_w + c
+\end{equation}
+```
+
+``\Delta a_w``is the difference between the water activity of the droplet, ``a_w``, and the water activity of ice at the same temperature, ``a_{w,ice}(T)``. From [Koop2002](@cite), 
+```math
+\begin{equation}
+  a_w = \frac{p_{sol}}{p_{sat}}
+\end{equation}
+```
+```math
+\begin{equation}
+  a_{w,ice} = \frac{p_{i,sat}}{p_{sat}}
+\end{equation}
+```
+
+where ``p`` is saturated vapor pressure of water above solution, pure liquid water, and ice for subscripts ``sol``, ``sat``, and ``i,sat`` respectively. ``p_{sol}`` is determined in mbar using a parameterization for supercooled, binary ``H_2SO_4/H_2O`` solution from [Luo1995](@cite) which is valid for 185K < T < 235K:
```

Comment:
`where ``p`` is saturated vapor pressure of water above solution, pure liquid water, and ice for subscripts ``sol``, ``sat``, and ``i,sat`` respectively.` - This one os a little confusing. Could you rephrase it?